src/main/java/org/mitre/synthea/modules/LifecycleModule.java

```diff
@@ -1070,6 +1078,7 @@ public static void inventoryAttributes(Map<String,Inventory> attributes) {
     Attributes.inventory(attributes, m, QUIT_SMOKING_PROBABILITY, true, false, null);
     Attributes.inventory(attributes, m, Person.RACE, true, false, null);
     Attributes.inventory(attributes, m, Person.SMOKER, true, false, "Boolean");
+    Attributes.inventory(attributes, m, Person.DEATHDATE, true, false, null);
```

Comment:
the DEATHDATE attribute gets set to be a timestamp so we could put an example value here
```suggestion
    Attributes.inventory(attributes, m, Person.DEATHDATE, true, false, "1046327126000");
```

Reply:
Done